### PR TITLE
Remove ExternalArgAddress

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp [hunk 1]
```diff
@@ -981,10 +981,22 @@ namespace Tensile
         {
             if(internalArgsSupport.useUniversalArgs)
             {
-                kernelArgs<T_Debug, false>(problems.size(), 1, rv.args, problems[0].getParams());
+                int argType = 1;
```
Comment (jichangjichang) [inline tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp]: Could we use enum for ArgType?

Comment (KKyang) [reply tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp]: Sure

File: tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp [hunk 2]
```diff
@@ -645,7 +652,8 @@ namespace Tensile
 
         if constexpr(insertKernelArgs)
             if(!internalArgsSupport.useUniversalArgs)
-                kernelArgs<T_Debug, true>(0, 0, args, problem.getParams());
+                kernelArgs<T_Debug, true>(
+                    0, (uint32_t)KERNELARGTYPE::NORMAL, args, problem.getParams());
```
Comment (jichangjichang) [inline tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp:656]: better to case at line#767, and set parameter type as KERNELARGTYPE at line#759.

**Review by jichangjichang — APPROVED** [review]
lgtm

**Review by vin-huang — APPROVED** [review]

**Pull request merged**